gz_ros2_control/include/gz_ros2_control/gz_system_interface.hpp

```diff
@@ -13,22 +13,28 @@
 // limitations under the License.
 
 
-#ifndef IGN_ROS2_CONTROL__IGN_SYSTEM_INTERFACE_HPP_
-#define IGN_ROS2_CONTROL__IGN_SYSTEM_INTERFACE_HPP_
+#ifndef GZ_ROS2_CONTROL__GZ_SYSTEM_INTERFACE_HPP_
+#define GZ_ROS2_CONTROL__GZ_SYSTEM_INTERFACE_HPP_
 
 #include <map>
 #include <memory>
 #include <string>
 #include <vector>
 
+#ifdef GZ_HEADERS
+#include <gz/sim/System.hh>
+#define GZ_SIM_NAMESPACE gz::sim::
+#else
 #include <ignition/gazebo/System.hh>
+#define GZ_SIM_NAMESPACE ignition::gazebo::
+#endif
 
 #include <hardware_interface/system_interface.hpp>
 #include <hardware_interface/types/hardware_interface_type_values.hpp>
 
 #include <rclcpp/rclcpp.hpp>
 
-namespace ign_ros2_control
+namespace gz_ros2_control
```

Comment:
This looks like an installed header and this change here would break API. Do we have a policy about that, especially if we plan to backport this PR?

If we don't expect others to use this API, should we stop installing the headers?

Reply:
if someone wants to use this headers I would like to force them to use the new API

Reply:
So, I guess the question is do we plan to backport this to older branches? If so, existing code would fail to compile.

Reply:
since there is a Humble release already, we probably can not backport anymore without breaking API. Rolling can break anyway.